apps/docs/pages/guides/ai/integrations/roboflow.mdx

```diff
@@ -0,0 +1,252 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'ai-integration-roboflow',
+  title:
+    'Roboflow',
+  subtitle:
+    'Learn how to integrate Supabase with Roboflow, a tool for running fine-tuned and foundation vision models.',
+  breadcrumb: 'AI Integrations',
+}
+
+This guide will walk you through a basic example using [Roboflow Inference](https://inference.roboflow.com) to run inference on fine-tuned and foundation models. We will run inference and save predictions using an object detection model and [CLIP](https://github.com/openai/CLIP).
```

Comment:
```suggestion
This guide will walk you through a basic example using [Roboflow Inference](https://inference.roboflow.com) to run inference on fine-tuned and foundation models. You will run inference and save predictions using an object detection model and [CLIP](https://github.com/openai/CLIP).
```

Minor nit: in our [style guide](https://github.com/supabase/supabase/blob/master/apps/docs/CONTRIBUTING.md) we prefer `you` over `we`